t/mojo/morbo.t

```diff
@@ -136,6 +136,49 @@ subtest 'Broken symlink' => sub {
   ok !$warned, 'no warnings';
 };
 
+subtest 'Randomized Request IDs' => sub {
+  my @ids;
+  my $src = <<'EOF';
+use Mojolicious::Lite;
+
+app->log->level('fatal');
+
+# Return "request_id"
+get '/request_id' => sub {
+  my $c = shift;
+  $c->render( text => $c->req->request_id );
+};
+
+app->start;
+EOF
+
+  # Fetch initial request_ids
+  $script->spurt($src);
+  sleep 3;
```

Comment:
Nevermind, I realized this is a developer test.

Reply:
I still run those tests and i hate waiting...

Reply:
FWIW, I just copied the `sleep 3` from the earlier tests in that file...  If there's an easier way to reliably block until Morbo reloads/restarts, I am absolutely, without a doubt, 100% all for it!

Like, if there were some kind of `*after*_server_start` hook that I could `await` on, or something?  (Does that even make sense?  Create a promise, start/restart the server, await, and then `resolve` it in a hook/handler?) I'm sure the devil is in the details; i.e., making it work for all the various `Mojo::Server::*` implementations surely involves voodoo that I can't even begin to fathom — otherwise I'm sure somebody would have knocked it by now!

† And isn't it great how, what seems like a lifetime ago, I remember `LOAD 1,1`-ing Commodore 64 games from a Cassette Tape in **less than \*10\* minutes!**  ;-D

Reply:
I've changed the existing tests in the meantime to make them faster.